src/lib.rs

```diff
@@ -109,18 +191,18 @@ pub(crate) struct Outbound {
 }
 
 impl Outbound {
-    #[inline(always)]
+    #[inline]
```

Comment:
The compiler has more information about when inlining is appropriate. `inline(always)` can actually slow down programs by causing the code section to grow in size, reducing the efficiency of your CPU's instruction cache. LLVM is usually pretty good about reasoning about where that line between instruction prefetcher-friendliness and instruction cache-friendliness is.

Reply:
Good to know, I saw that they also are doing away with it.